pages/ScheduleDetailPage.js

```diff
@@ -53,8 +53,7 @@ const styles = StyleSheet.create({
     },
     container: {
         flex: 1,
-        backgroundColor: '#1a1d32',
-        paddingLeft: 10
```

Comment:
Why did you take out the padding?

Reply:
The whole container was off-center, at least on Android

Reply:
Oh uh, I meant it for the listview because it was too close to the edge, just make sure it's not too close to the edge...

Reply:
Alright, I'll move the rows a bit more before merging